lib/rules/web/admin_console/4.2/cluster_settings.xyaml

```diff
@@ -0,0 +1,106 @@
+# check cluster settings tab links
+check_cluster_settings_tab_links:
+  params:
+    text: Overview 
+    link_url: /settings/cluster/
+  action: check_link_and_text
+  params:
+    text: Cluster Operators 
+    link_url: /settings/cluster/clusteroperators
+  action: check_link_and_text
+  params:
+    text: Global Configuration 
+    link_url: /settings/cluster/globalconfig
+  action: check_link_and_text
+# check version table box section
+check_table_header:
+  elements:
+  - selector:
+      xpath: //.[contains(@class, 'co-detail-table') and contains(text(), '<text>')]
+check_version_table:
+  params:
+    text: Channel 
+  action: check_table_header
+  params:
+    text: Last Completed Version 
+  action: check_table_header
+  params:
+    text: Update Status 
+  action: check_table_header
+  elements:
+  - selector:
+      xpath: //div[contains(@class, 'co-detail-table__section')][1]//button[contains(@class, 'btn btn-link')]
+  - selector:
+      xpath: //div[contains(@class, 'co-detail-table__section')][2]//div[contains(text(), '4.2')]
+  - selector:
+      xpath: //div[contains(@class, 'co-detail-table__section')][3]//div[contains(text(), .)]
```

Comment:
what does this do?

Reply:
to ensure there is something in the value at my first thought